HISTORY.md

```diff
@@ -0,0 +1,41 @@
+# Pegasus Change Log
+
+## 2.5.0
+
+### New Features
+* Add a new ACL based on Apache Ranger to provide fine-grained access control to global-level, database-level and table-level resources. On the other hand, it is also compatible with the old coarse-grained ACL. [#1054](https://github.com/apache/incubator-pegasus/issues/1054)
+* Add support to query and update table-level rocksdb options dynamically, where currently only `num_levels` and `write_buffer_size` are supported; other options would be added gradually, if necessary. [#1511](https://github.com/apache/incubator-pegasus/pull/1511)
+* Pegasus could be built on more platforms: Ubuntu 22.04/Clang 14, M1 MacOS. [#1350](https://github.com/apache/incubator-pegasus/pull/1350), [#1094](https://github.com/apache/incubator-pegasus/pull/1094)
+* Pegasus could be developed and built in a docker environment, which is more friendly to newbies. [#1544](https://github.com/apache/incubator-pegasus/pull/1544)
+* Provide a RESTful service to query immediate values of metrics from new framework, where retrievals according to filters given by users, such as entity types, ids, attributes and metric names, are also supported. [#1206](https://github.com/apache/incubator-pegasus/issues/1206)
```

Comment:
Let's leave this point to the next release after the whole work finished, because there is no metrics can be queried via RESTful API currently.